census/tf-keras/trainer/model.py

```diff
@@ -54,31 +54,29 @@ def input_fn(features, labels, shuffle, num_epochs, batch_size):
 def create_keras_model(input_dim, learning_rate):
   """Creates Keras Model for Binary Classification.
 
+  The single output node + Sigmoid activation makes this a Logistic Regression.
+
   Args:
     input_dim: How many features the input has
     learning_rate: Learning rate for training
 
   Returns:
     The compiled Keras model (still needs to be trained)
   """
-  model = tf.keras.Sequential()
-  model.add(
-      tf.keras.layers.Dense(
-          100,
-          activation=tf.nn.relu,
-          kernel_initializer='uniform',
-          input_shape=(input_dim,)))
-  model.add(tf.keras.layers.Dense(75, activation=tf.nn.relu))
-  model.add(tf.keras.layers.Dense(50, activation=tf.nn.relu))
-  model.add(tf.keras.layers.Dense(25, activation=tf.nn.relu))
-  # The single output node and Sigmoid activation makes this a Logistic
-  # Regression.
-  model.add(tf.keras.layers.Dense(1, activation=tf.nn.sigmoid))
+  l = tf.keras.layers
```

Comment:
Nit: The point of this is just to save characters in the next few lines, right? I don't love the [single character `l` variable name](http://google.github.io/styleguide/pyguide.html#3161-names-to-avoid). Would `Dense = tf.keras.layers.Dense` work just as well? (and then just use `Dense` instead of `l.Dense`).

Reply:
Thanks, updated. I think this has become a bad standard in TF-Keras